react-d3-chart/src/components/Axis/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useRef } from 'react';
+import * as d3 from 'd3';
+import styled from 'styled-components';
+
+const Text = styled.text`
+  fill: black;
+  font-family: sans-serif;
+  font-size: 10px;
+`;
+
+interface Props {
+  scale: d3.ScaleLinear<number, number>;
+  type: 'Top' | 'Right' | 'Bottom' | 'Left';
+  x: number;
+  y: number;
+  label?: string;
+}
+
+const Axis: React.FC<Props> = ({ x, y, label, scale, type }) => {
```

Comment:
How are you planning to control axis changes on zoom/brush events?

Reply:
Not sure haven't looked at either yet. Wanted to open a PR for the basic graph + bisector / scanner